src/subscription_manager/managercli.py

```diff
@@ -539,7 +540,7 @@ def main(self, args=None):
 
             if return_code is not None:
                 return return_code
-        except X509.X509Error, e:
+        except (CertificateException, ssl.SSLException), e:
```

Comment:
might as well make this `(CertificateException, ssl.SSLException) as e:` for Python 3 future-proofing.